app/models/cohort.rb

```diff
@@ -1,5 +1,5 @@
 class Cohort < ApplicationRecord
-  has_many :applications, dependent: :destroy
+  has_many :applications, -> { order('state DESC, id') }, dependent: :destroy
```

Comment:
I wonder if we could put the sorting logic in the controller layer so we don't sort everywhere we make this call. WDYT?

Feel free to punt

Reply:
@srt32 I actually was thinking of a better solution as I did that for that reason and I think i came up with something that works and is just for this view. Let me know what you think